src/Native/ObjWriter/objwriter.cpp

```diff
@@ -837,3 +838,56 @@ ObjectWriter::GetMemberFunctionId(const MemberFunctionIdTypeDescriptor& MemberId
     return TypeBuilder.GetMemberFunctionId(MemberIdDescriptor);
 }
 
+void
+ObjectWriter::EmitFnStart() {
```

Comment:
Should this be called `EmiARMtFnStart` ? It seems to be ARM specific looking at the implementation.